decidim-core/spec/system/user_timeline_spec.rb

```diff
@@ -57,6 +57,21 @@ def resource_types
     create(:action_log, action: "publish", visibility: "all", resource: resource3, organization: organization)
   end
 
+  let!(:resource_types) do
```

Comment:
is this being used?

Reply:
I changed it so now uses a specific array with the resource_types requested. The spec uses that variable